package.json

```diff
@@ -4,16 +4,17 @@
   "description": "Kinto Web Administration Console in React.js",
   "type": "module",
   "scripts": {
-    "build": "vite build",
+    "build": "npm run generate-version && vite build",
```

Comment:
This seems like a job for a `Makefile`, which can come in another PR. It'd be nice to add our common targets like `build`, `start`, `lint`, `format`, etc. while we're at it.

Reply:
I don't think I've ever seen a Makefile for a pure react project before. I'm not opposed to this, but it might cause friction with pure UI devs. We should ask around a bit.